FastApi/app/routers/image.py

```diff
@@ -0,0 +1,58 @@
+import pathlib
+
+from fastapi import FastAPI, Response, requests, status, HTTPException, Depends, APIRouter
+from fastapi import File, UploadFile, Request
+from fastapi.responses import FileResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.exceptions import HTTPException
+
+from sqlalchemy.orm import Session
+from typing import Annotated, List, Optional
+
+from sqlalchemy import func
+from .. import models
+from ..database import get_db
+
+
+router = APIRouter(prefix="/images", tags=["Images"])
+
+image_folder_path="/home/dev-222/FOLDAAA/Praksa/Python/FastApi/static/Images/"
+url_folder_path="127.0.0.1:8000/static/Images/"
+
```

Comment:
Is this file necessary? if it is, then add file validation here as well and fix error handling